src/test/java/seedu/address/logic/commands/AddFavouriteCommandTest.java

```diff
@@ -59,14 +59,20 @@ public void execute_invalidIndex_throwsCommandException() {
     }
 
     @Test
-    public void execute_favouriteContact_throwsCommandException() {
+    public void execute_alreadyFavourite_success() {
```

Comment:
Do you think it would be appropriate if I named this test case `execute_alreadyFavourite_successWithWarning`?

Reply:
Sure